app/views/comments/new.html.erb

```diff
@@ -0,0 +1 @@
+<!-- Empty template, nothing to show -->
```

Comment:
- [ Required ] I think it would be better if you follow the project requirements for this milestone, according to the project requirements the application should have a form in the `views/comments/new.html.erb` file which accepts input for creating `comments`. This means you should make sure that as much as a user can add a comment from the `posts show` page, when a user navigates to `GET /posts/:post_id/comments/new` the user should be able to add a comment to a post through a form also. Kindly implement this.

*[Hint] You can use the form on the posts show page*